web-api/src/persistence/postgres/connections/getAllWebSocketConnections.ts

```diff
@@ -0,0 +1,7 @@
+import { getDbReader } from '@web-api/database';
+import { Connection } from '@web-api/notifications/sendNotificationToConnection';
+
+export const getAllWebSocketConnections = async (): Promise<Connection[]> =>
```

Comment:
This is where we could use a `ConnectionKysely`. The reason this works here without that is because the TS type for Connection and the type for the data in the database basically intersect entirely. Not saying that this is necessarily better--on the one hand, it is nice to have fewer types; on the other, it is less consistent with other Postgres work.